### PR TITLE
MAUI-1448  Prepare UG documents for Maui linear gauge control and radial gauge new features

### PR DESCRIPTION
**Pull request by MuneeshKumarG**
UG provided for below sections. 

1. SfLinearGauge control
2. SfRadialGauge -> ShapePointer, ContentPointer

### CODE REVIEW

File: MAUI/LinearGauge/animation.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+title: Animation in .NET MAUI Linear Gauge control | Syncfusion
+description: Learn here all about adding and customizing animation of Syncfusion .NET MAUI Linear Gauge (SfLinearGauge) control and more.
+platform: maui
+control: SfLinearGauge
+documentation: ug
+---
+
+# Animation in .NET MAUI Linear Gauge (SfLinearGauge)
+
+All Linear Gauge elements such as axis along with ticks and labels, range, bar pointer, shape marker pointer and content marker pointer can be animated separately.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/animation.md:12]: All Linear Gauge elements including the axis, ticks and labels, range, bar pointer, shape marker pointer, and content marker pointer, can be animated separately.

File: MAUI/LinearGauge/animation.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Animate axis in linear gauge](images/animation/animation_axis.gif)
+
+## Animate range
+
+The [`EnableRangeAnimation`]() and [`AnimationDuration`]() properties in [`SfLinearGauge`]() is used to  animate the scale along with the ticks and labels. The range will be have a fade-in with opacity animation when this [`EnableRangeAnimation`]() is set to true. By default, the [`EnableRangeAnimation`]() is set to false.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/animation.md:40]: The [`EnableRangeAnimation`]() and [`AnimationDuration`]() properties in [`SfLinearGauge`]() is used to animate the scale along with the ticks and labels. The range will have a fade-in with opacity animation when this [`EnableRangeAnimation`]() is set to true. By default, the [`EnableRangeAnimation`]() is set to false.

check the spacing.

File: MAUI/LinearGauge/animation.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Animate range in linear gauge](images/animation/animation_range.gif)
+
+## Pointer animation
+
+The animation behavior is common for all the three pointers in Linear Gauge - shape, content and bar pointer.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/animation.md:64]: The animation behavior is common for all the three pointers in Linear Gauge - shape, content, and bar pointer.

File: MAUI/LinearGauge/animation.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+{% endtabs %}
+
+![Animate range in linear gauge](images/animation/animation_range.gif)
+
+## Pointer animation
+
+The animation behavior is common for all the three pointers in Linear Gauge - shape, content and bar pointer. 
+
+All the above three pointers have the below properties for animation.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/animation.md:66]: All the above three pointers have the following properties for the animation:

File: MAUI/LinearGauge/animation.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+
+The animation behavior is common for all the three pointers in Linear Gauge - shape, content and bar pointer. 
+
+All the above three pointers have the below properties for animation. 
+
+*  [`EnableAnimation`]() - Enable or disable the animation for pointer. The default value is `false`.
+*  [`AnimationDuration`]() - Sets the animation duration. The default value is 1000.
+*  [`AnimationEasing`]() - Sets the animation type. 
+
+The [`AnimationEasing`]() supports the below animations. The default animation type is `Easing.Linear`.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/animation.md:72]: The [`AnimationEasing`]() supports the following animations. The default animation type is `Easing.Linear`.

File: MAUI/LinearGauge/animation.md [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+
+* `BounceIn`
+
+* `SpringIn`
+
+* `SpringOut`
+
+### Animate bar pointer
+
+The below code example demonstrates updating the animation for bar pointer.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/animation.md:98]: The following code example demonstrates how to update the animation for bar pointer.

File: MAUI/LinearGauge/animation.md [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+        });
+	this.Content = gauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Animate marker pointers (Shape and Content Pointers)
+
+Both the shape and content marker pointers will have the same set of properties and behave similarly for animation. So we have demonstrated the [`LinearShapePointer`]() only but the same is applicable for [`LinearContentPointer`]() too.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/animation.md:128]: So, demonstrated the [`LinearShapePointer`]() only but the same is applicable for [`LinearContentPointer`]() too.

File: MAUI/LinearGauge/bar-pointer.md [hunk 8]
```diff
@@ -0,0 +1,307 @@
...
+title: Bar Pointer in .NET MAUI Linear Gauge control | Syncfusion
+description: Learn here all about adding and customizing Bar Pointer of Syncfusion .NET MAUI Linear Gauge (SfLinearGauge) control and more.
+platform: maui
+control: SfLinearGauge
+documentation: ug
+---
+
+# Bar Pointer in .NET MAUI Linear Gauge (SfLinearGauge)
+
+A bar pointer is an accenting line or shaded background that can be placed on a Linear Gauge to mark any current value in the axis track. The bar pointers always start from the minimum value of the axis and end with the specified value. So the [`Value`]() property is a required parameter for creating a bar pointer.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/bar-pointer.md:12]: So, the [`Value`]() property is a required parameter for creating a bar pointer.

File: MAUI/LinearGauge/bar-pointer.md [hunk 9]
```diff
@@ -0,0 +1,307 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Initialize linear gauge for bar pointer](images/bar-pointer/default_bar_pointer.PNG)
+
+## Customize bar pointer thickness
+
+The thickness can be changed by the [`PointerSize`]() property of the bar pointer. The following code sample demonstrates the same.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/bar-pointer.md:47]: The thickness can be changed using the [`PointerSize`]() property of the bar pointer.

File: MAUI/LinearGauge/bar-pointer.md [hunk 10]
```diff
@@ -0,0 +1,307 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Change the bar pointer thickness](images/bar-pointer/bar_thickness.PNG)
+
+## Customize edge style
+
+The edge style can be changed with the [`CornerStyle`]() property of the bar pointer. The edge style can be any of the `StartCurve`, `EndCurve`, `BothCurve`, and `BothFlat` options.The default value is `BothFlat`.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/bar-pointer.md:79]: The edge style can be changed using the [`CornerStyle`]() property of the bar pointer. The edge style can be any of the `StartCurve`, `EndCurve`, `BothCurve`, and `BothFlat` options. The default value is `BothFlat`.

check the spacing

File: MAUI/LinearGauge/bar-pointer.md [hunk 11]
```diff
@@ -0,0 +1,307 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Customize linear gauge for bar pointer position](images/bar-pointer/bar_pointer_change_position.PNG)
+
+## Customize the offset
+
+In addition to position the bar pointer, it is also possible to change the offset of the bar pointer. The [`Offset`]() is the distance from the axis and the cross positioned elements will not get affected by the [`Offset`]() value. The following  code sample demonstrates how to change the offset value of the bar pointer.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/bar-pointer.md:148]: The [`Offset`]() value is the distance from the axis and it has no effect on cross-positioned elements.

File: MAUI/LinearGauge/bar-pointer.md [hunk 12]
```diff
@@ -0,0 +1,307 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Apply radial gradient to bar pointer](images/bar-pointer/gradient_bar.PNG)
+
+## Child support
+
+By using [`Child`]() property of bar pointer, you can provide a child for bar pointer. The child content is used to add any UI content like text or images, inside the bar pointer to improve readability.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/bar-pointer.md:257]: By using the [`Child`]() property of bar pointer, you can provide a child for the bar pointer. To improve readability, the child content is used to add any UI content, such as text or images, inside the bar pointer.

File: MAUI/LinearGauge/content-marker-pointer.md [hunk 13]
```diff
@@ -0,0 +1,202 @@
...
+title: Content Marker Pointer in .NET MAUI Linear Gauge control | Syncfusion
+description: Learn here all about adding and customizing Content Marker Pointer of Syncfusion .NET MAUI Linear Gauge (SfLinearGauge) control and more.
+platform: maui
+control: SfLinearGauge
+documentation: ug
+---
+
+# Content Marker Pointer in .NET MAUI Linear Gauge (SfLinearGauge)
+
+The [`LinearContentPointer`]() in [`SfLinearGauge`]() allows to use any .NET MAUI content as marker pointer. The following code sample uses a [`Image`]() as marker pointer.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/content-marker-pointer.md:12]: The [`LinearContentPointer`]() in [`SfLinearGauge`]() allows you to use any .NET MAUI content as a marker pointer. The following code sample uses a [`Image`]() as a marker pointer.

File: MAUI/LinearGauge/content-marker-pointer.md [hunk 14]
```diff
@@ -0,0 +1,202 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Change content pointer position](images/content-pointer/content_pointer_position.PNG)
+
+## Change the offset
+
+In addition to position the content marker pointer, it is also possible to change the offset of the shape pointer. The [`OffsetX`]() and [`OffsetY`]() are the distance from the axis. The cross positioned elements will not get affected by the [`OffsetX`]() and [`OffsetY`]() values. The following code sample demonstrates how to change the offset value of the shape pointer.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/content-marker-pointer.md:153]: cross positioned----->cross-positioned

File: MAUI/LinearGauge/getting-started.md [hunk 15]
```diff
@@ -0,0 +1,231 @@
...
+
+# Getting Started with .NET MAUI Linear Gauge
+
+This section explains the steps required to add the [`.NET MAUI Linear Gauge`]() control and its elements such as axis, range, and pointer. This section covers only basic features needed to get started with Syncfusion linear gauge control.
+
+## Creating an application using the .NET MAUI Linear Gauge
+
+* Create a new .NET MAUI application in the Visual Studio.
+
+* Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add SfLinearGauge to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Gauges] and then install it.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/getting-started.md:18]: Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add `SfLinearGauge` to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Gauges], and then install it.

File: MAUI/LinearGauge/getting-started.md [hunk 16]
```diff
@@ -0,0 +1,231 @@
...
+SfLinearGauge gauge = new SfLinearGauge();
+this.Content = gauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Register the handler
+
+Syncfusion.Maui.Core nuget is a dependent package for all Syncfusion controls of .NET MAUI. In the MauiProgram.cs file register the handler for Syncfusion core.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/getting-started.md:59]: Syncfusion.Maui.Core NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the MauiProgram.cs file, register the handler for Syncfusion core.

File: MAUI/LinearGauge/getting-started.md [hunk 17]
```diff
@@ -0,0 +1,231 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Linear Gauge with Range](images/getting-started/maui-linear-gauge-with-range.PNG)
+
+## Add pointer to the linear gauge
+
+Pointers contains a list of pointer elements, where you can add any number of gauge pointers such as [`BarPointer`](), [`ShapePointer`]() and [`ContentPointer`]() inside the scale to indicate the value.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/getting-started.md:123]: Pointers contain a list of pointer elements, in which you can add any number of gauge pointers such as [`BarPointer`](), [`ShapePointer`](), and [`ContentPointer`]() inside the scale to indicate the value.

File: MAUI/LinearGauge/mirror-linear-gauge.md [hunk 18]
```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: Mirror in .NET MAUI Linear Gauge control | Syncfusion
+description: Learn here about mirroring the Syncfusion .NET MAUI Linear Gauge (SfLinearGauge) control with IsMirrored property
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/mirror-linear-gauge.md:4]: Learn how to use the `IsMirrored` property to mirror the Syncfusion.NET MAUI Linear Gauge (SfLinearGauge) control.

File: MAUI/LinearGauge/mirror-linear-gauge.md [hunk 19]
```diff
@@ -0,0 +1,38 @@
...
+platform: maui
+control: SfLinearGauge
+documentation: ug
+---
+
+# Mirrored in .NET MAUI Linear Gauge (SfLinearGauge)
+
+The [`IsMirrored`]() property in [`SfLinearGauge`]() mirrors all the Gauge elements in the `SfLinearGauge`. The following code sample demonstrates how to setting the [`IsMirrored`]() property.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/mirror-linear-gauge.md:12]: The [`IsMirrored`]() property in [`SfLinearGauge`]() mirrors all Gauge elements in the `SfLinearGauge`. The following code sample demonstrates how to set the [`IsMirrored`]() property.

File: MAUI/LinearGauge/overview.md [hunk 20]
```diff
@@ -0,0 +1,28 @@
...
+# .NET MAUI Linear Gauge (SfLinearGauge) Overview
+
+Syncfusion .NET MAUI Linear Gauge is a data visualization control to display data on a linear scale. Use this control to craft high-quality mobile app user interfaces.
+
+![Overview .NET MAUI linear gauge](images/basic_elements.PNG)
+
+## Key Features
+
+* **Orientation** - The Linear Gauge can be set to vertical or horizontal orientation.
+* **Scale** - The scale is a set of values can be plotted. An scale can be customized by changing the thickness and edge styles and you can also inverse the scale.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/overview.md:19]: The scale is a set of values that can be plotted. You can customize a scale by changing the thickness and edge styles, and you can also inverse the scale.

File: MAUI/LinearGauge/overview.md [hunk 21]
```diff
@@ -0,0 +1,28 @@
...
+* **Labels and Ticks** - The axis elements such as labels, major ticks, and minor ticks can be customized to different styles.
+* **Ranges** - A range is a visual element that helps you to quickly visualize where a range falls on the scale track. Multiple ranges with different styles can be added to the Linear Gauge.
+* **Pointers** - A pointer is used to indicate a specific value on an scale. The gauge has three types of pointers: shape marker pointer, content marker pointer, and bar pointer. All the pointers can be customized as needed and you can also add multiple pointers in the Linear Gauge.
+* **Mirror Gauge** - The Linear Gauge can be mirrored. When the [`IsMirrored`]() property is true, all the gauge elements will be rendered in mirror effect. 
+* **Animation** - Animate the gauge elements in a visually appealing way, when they are load, or when their values are changed.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/overview.md:24]: Animate the gauge elements in a visually appealing manner when they are loaded or their values are changed.

File: MAUI/LinearGauge/pointers.md [hunk 22]
```diff
@@ -0,0 +1,186 @@
...
+
+# Pointers in .NET MAUI Linear Gauge (SfLinearGauge)
+
+The pointer is used to indicate values on a scale. The Linear Gauge control has three types of pointers:
+
+[`Bar pointer`]()
+[`Shape Pointer`]()
+[`Content pointer`]()
+
+All the pointers can be customized as needed. You can add multiple pointers to the gauge to point multiple values on the same scale. The value of the pointer is set using the Value property.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/pointers.md:18]: The value of the pointer is set using the `Value` property.

Please highlight the property names

File: MAUI/LinearGauge/pointers.md [hunk 23]
```diff
@@ -0,0 +1,186 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Drag offset
+
+The [`DragOffset`]() property is used to specify the outer dragging offset for pointer. When the pointer is pressed close to the pointer, this value helps in dragging the pointer. The default value of [`DragOffset`]() is 15d.
+
+For example, if the value of [`DragOffset`]() is 20, the pointer will be dragged while interact with pointer around 20 pixel.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/pointers.md:96]: if the value of [`DragOffset`]() is 20, the pointer will be dragged while interacting with another pointer around 20 pixels.

File: MAUI/LinearGauge/pointers.md [hunk 24]
```diff
@@ -0,0 +1,186 @@
...
+
+{% endtabs %}
+
+## Drag offset
+
+The [`DragOffset`]() property is used to specify the outer dragging offset for pointer. When the pointer is pressed close to the pointer, this value helps in dragging the pointer. The default value of [`DragOffset`]() is 15d.
+
+For example, if the value of [`DragOffset`]() is 20, the pointer will be dragged while interact with pointer around 20 pixel. 
+
+If the value of [`DragOffset`]() is 0, the pointer will be dragged while interact with exact pointer bounds alone.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/pointers.md:98]: If the value of [`DragOffset`]() is 0, the pointer will be dragged while interacting with exact pointer bounds alone.

File: MAUI/LinearGauge/scale.md [hunk 25]
```diff
@@ -0,0 +1,362 @@
...
+description: Learn here all about adding and customizing Axis of Syncfusion .NET MAUI Linear Gauge (SfLinearGauge) control and more.
+platform: maui
+control: SfLinearGauge
+
+documentation: ug
+---
+
+# Axis in .NET MAUI Linear Gauge (SfLinearGauge)
+
+The Linear Gauge axis is a scale where a set of values can be plotted. An axis can be customized by changing the thickness, color and edge styles. Axis elements such as labels and ticks can also be easily customized and you can also inverse the axis.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/scale.md:13]: The Linear Gauge axis is a scale on which a set of values can be plotted. An axis can be customized by changing the thickness, color, and edge styles. Axis elements such as labels and ticks can also be easily customized and axis can be inversed.

File: MAUI/LinearGauge/scale.md [hunk 26]
```diff
@@ -0,0 +1,362 @@
...
+
+## Default axis
+
+By default axis will have the [`Minimum`]() axis value as 0 and the [`Maximum`]() axis value as 100. Without any changes the default axis of the Linear Gauge will be displayed as follows.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/scale.md:17]: By default, axis will have the [`Minimum`]() axis value as 0 and the [`Maximum`]() axis value as 100. Without any changes, the default axis of the Linear Gauge will be displayed as follows.

File: MAUI/LinearGauge/scale.md [hunk 27]
```diff
@@ -0,0 +1,362 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Initialize linear gauge for axis](images/getting-started/default_linear_gauge.PNG)
+
+## Customize minimum and maximum axis values
+
+The [`Minimum`]() and [`Maximum`]() properties of a Linear Gauge can be used to customize the axis values. In the below code snippet the axis is customized to have the [`Minimum`]() value of -50 to [`Maximum`]() value of 50. The axis values are displayed by the labels. Customizing these label styles are further explained in next topics.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/scale.md:40]: In the following code sample, the axis is customized to have the [`Minimum`]() value of -50 to [`Maximum`]() value of 50.

File: MAUI/LinearGauge/scale.md [hunk 28]
```diff
@@ -0,0 +1,362 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Update orientation for axis scale](images/axis/linear_gauge_orientation.PNG)
+
+## Customize axis track style
+
+The linear axis track can be customized using the [`LineStyle`]() property. The [`LineStyle`]() has the following properties.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/scale.md:91]: The [`LineStyle`]() has the following properties:

File: MAUI/LinearGauge/scale.md [hunk 29]
```diff
@@ -0,0 +1,362 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Change axis track thickness in linear gauge](images/axis/axis_thickness.PNG)
+
+## Apply solid color
+
+The below code snippet sets solid colors to the axis track.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/scale.md:128]: The following code sample sets solid colors to the axis track.

File: MAUI/LinearGauge/scale.md [hunk 30]
```diff
@@ -0,0 +1,362 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Hide linear gauge axis track](images/axis/hide_axis_track.PNG)
+
+## Customize axis values
+
+Linear gauge allows you to display a set of values along with a custom axis based on your business logic by using the [`GenerateVisibleLabels`](),[`ValueToFactor`]() and [`FactorToValue`]() callbacks.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/scale.md:303]: [`GenerateVisibleLabels`](),[`ValueToFactor`](), and [`FactorToValue`]() callbacks.

File: MAUI/LinearGauge/shape-marker-pointer.md [hunk 31]
```diff
@@ -0,0 +1,277 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Change shape pointer elevation](images/shape-pointer/pointer_shadow.PNG)
+
+## Change marker alignment
+
+The marker pointer alignment can be changed by the [`Alignment`]() property of [`LinearShapePointer`]().The available marker pointer alignments are `Start`, `End`, and `Center`.
```
Comment (laksmipriyamurugesan) [inline MAUI/LinearGauge/shape-marker-pointer.md:182]: Check the spacing

File: MAUI/RadialGauge/content-pointer.md [hunk 32]
```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: Content Pointer in .NET MAUI Radial Gauge control | Syncfusion
+description: Learn here all about Content Pointer feature of Syncfusion .NET MAUI Radial Gauge control with image, text pointer support and more.
```
Comment (laksmipriyamurugesan) [inline MAUI/RadialGauge/content-pointer.md:4]: Learn here all about Content Pointer feature of the Syncfusion .NET MAUI Radial Gauge control, including image, text pointer support and more.

File: MAUI/RadialGauge/content-pointer.md [hunk 33]
```diff
@@ -0,0 +1,176 @@
...
+
+{% endtabs %}
+
+![.NET MAUI Radial Gauge Content Pointer](images/marker-pointers/maui-radial-gauge-content-pointer.PNG)
+
+## Position customization
+
+The content pointer can be moved near or far from its actual position using the [`Offset`]() and [`OffsetUnit`]() properties.
+
+When you set [`OffsetUnit`]() to logical pixel, then the content pointer will be moved based on the logical pixel value. If you set [`OffsetUnit`]() to factor, then provided factor will be multiplied with the axis radius value, and then the pointer will be moved to corresponding value. The default value of [`OffsetUnit`]() is [`SizeUnit.Pixel`]().
```
Comment (laksmipriyamurugesan) [inline MAUI/RadialGauge/content-pointer.md:96]: When you set [`OffsetUnit`]() to logical pixel, the content pointer is moved based on the logical pixel value. If you set [`OffsetUnit`]() to factor, then provided factor will be multiplied by the axis radius value, and then the pointer will be moved to the corresponding value. The default value of [`OffsetUnit`]() is [`SizeUnit.Pixel`]().

File: MAUI/RadialGauge/shape-pointer.md [hunk 34]
```diff
@@ -0,0 +1,244 @@
...
+## Shape customization
+
+The shape pointer can be customized using the following properties:
+
+* [`Fill`]() – Allows you to customize the shape color.
+* [`ShapeHeight`]() – Allows you to specify the shape height.
+* [`ShapeWidth`]() – Allows you to specify the shape width.
+* [`Stroke`]() – Allows you to specify the border color for the shape.
+* [`BorderWidth`]() –  Allows you to specify the border width of the shape.
+* [`HasShadow`]() –  Allows you to specify the shadow of the shape.
```
Comment (laksmipriyamurugesan) [inline MAUI/RadialGauge/shape-pointer.md:68]: check the spacing.

File: MAUI/RadialGauge/shape-pointer.md [hunk 35]
```diff
@@ -0,0 +1,244 @@
...
+* [`OverlayFill`]() –  Allows you to specify the color for dragging overlay for the shape.
```
Comment (laksmipriyamurugesan) [inline MAUI/RadialGauge/shape-pointer.md:69]: Allows you to specify the color for the shapes' dragging overlay
 
 Extra spacing.

File: MAUI/RadialGauge/shape-pointer.md [hunk 36]
```diff
@@ -0,0 +1,244 @@
...
+* [`OverlayRadius`]() –  Allows you to specify the radius for dragging overlay for the shape.
```
Comment (laksmipriyamurugesan) [inline MAUI/RadialGauge/shape-pointer.md:70]: Extra spacing

Allows you to specify the radius for the shapes' dragging overlay.

File: MAUI/RadialGauge/shape-pointer.md [hunk 37]
```diff
@@ -0,0 +1,244 @@
...
+{% endtabs %}
+
+![Change shape pointer overlay](images/marker-pointer/pointer_overlay.PNG)
+
+## Marker position customization
+
+The marker pointer can be moved near or far from its actual position using the [`Offset`]() and [`OffsetUnit`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.MarkerPointer.html#Syncfusion_Maui_Gauges_MarkerPointer_OffsetUnit) properties. 
+
+When you set [`OffsetUnit`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.MarkerPointer.html#Syncfusion_Maui_Gauges_MarkerPointer_OffsetUnit) to pixel, the marker pointer will be moved based on the pixel value. If you set [`OffsetUnit`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.MarkerPointer.html#Syncfusion_Maui_Gauges_MarkerPointer_OffsetUnit) to factor, then provided factor will be multiplied with the axis radius value, and then the pointer will be moved to corresponding value. The default value of [`OffsetUnit`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.MarkerPointer.html#Syncfusion_Maui_Gauges_MarkerPointer_OffsetUnit) is [`SizeUnit.Pixel`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.SizeUnit.html#Syncfusion_Maui_Gauges_SizeUnit_Pixel).
```
Comment (laksmipriyamurugesan) [inline MAUI/RadialGauge/shape-pointer.md:207]: When you set [`OffsetUnit`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.MarkerPointer.html#Syncfusion_Maui_Gauges_MarkerPointer_OffsetUnit) to pixel, the marker pointer is moved based on the pixel value. If you set [`OffsetUnit`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.MarkerPointer.html#Syncfusion_Maui_Gauges_MarkerPointer_OffsetUnit) to factor, then provided factor will be multiplied by the axis radius value, and then the pointer will be moved to the corresponding value. The default value of [`OffsetUnit`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.MarkerPointer.html#Syncfusion_Maui_Gauges_MarkerPointer_OffsetUnit) is [`SizeUnit.Pixel`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.SizeUnit.html#Syncfusion_Maui_Gauges_SizeUnit_Pixel).

**Review by sheiksyedm — APPROVED** [review]

**Pull request merged**